.gitlab-ci.yml

```diff
@@ -859,6 +859,8 @@ test_backend_integration:
     - for repo in `integration/extra/release_tool.py -l docker`; do
         integration/extra/release_tool.py --set-version-of $repo --version pr;
       done
+    # Other dependencies
+    - tar -xf stage-artifacts/host-tools.tar ./mender-artifact && mv mender-artifact ./integration/backend-tests/mender-artifact
```

Comment:
I would perhaps argue that it should be installed in `/usr/local/bin`, for simplicity. But this is fine too, I'll let you decide what's best.